### PR TITLE
Added bootstrap (CSS / JS)

### PR DESCRIPTION
**Pull request by jmc317**

### CODE REVIEW

**Review by Kasleets — CHANGES_REQUESTED** [review]
This is the Sweet Alert pop-ups, not exactly bootstrap. We don't have this plugin installed, I believe it requires the files first. They provide us a script that we could add to the index.html

<script src="https://cdn.jsdelivr.net/npm/sweetalert2@11"></script>

More information available in:
https://sweetalert2.github.io/#download

File: Properties/launchSettings.json [hunk 1]
```diff
@@ -1,37 +1,47 @@
-﻿{
-  "iisSettings": {
```
Comment (Kasleets) [inline Properties/launchSettings.json:2]: Are those code blocks irrelevant that you removed them?

File: wwwroot/js/site.js [hunk 2]
```diff
@@ -1,7 +1,21 @@
-﻿// Please see documentation at https://docs.microsoft.com/aspnet/core/client-side/bundling-and-minification
```
Comment (Kasleets) [inline wwwroot/js/site.js:1]: Perhaps it would be still good to keep the documentation in the code?

**Review by Kasleets — COMMENTED** [review]
Two small annotations / questions.

**Review by Xia-Hui-Liu — APPROVED** [review]

**Comment by Kasleets** [comment]
Closing and removing for Johan's request.

**Pull request closed without merging**